### PR TITLE
Dzhuha/merge rvat 869 with dev

### PR DESCRIPTION
**Pull request by JugerJuger**

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/archex/ProfilePage.java [hunk 1]
```diff
@@ -25,9 +28,23 @@ public EditAvartarPage getEditAvatarPage () {
         chengeButton.click();
         return new EditAvartarPage(driver);
     }
-
+    
+	public EditProfilePage clickEdit(){
+		waitForVisibilityOfElement(edit, MAX_TIME_WAIT);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/ProfilePage.java:33]: Why not use the clickable condition?

File: src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		driver.manage().window().maximize();
+		pageProfile = new HeadForGuestUserPage(driver).clickLeftNavButton().clickLogIn()
+				.setLoginField(Resources.LogInMessage.USER_EMAIL).setPasswordField(Resources.LogInMessage.USER_PASSWORD).clickLogInButton().clickLeftNavButton()
+				.clickProfile();
+		page = pageProfile.clickEdit();
+	}
+
+	@Test
+	public void verifyNameOfEditButton() {
+		Assert.assertTrue(pageProfile.getEditButton().getText().contains("Edit"));
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java:36]: Why contains instead of equals?

Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java:36]: Move to const

**Review by dmatsev — APPROVED** [review]
with questions

File: src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java (hunk 2 above)
Comment (JugerJuger) [reply src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java:36]: I just moved written test to page object, and tried minimised change of logic. It was assertTrue before me it was at demo, so i thought that author had some extra reason for this.

**Pull request merged**